metrics/integration_test.go

```diff
@@ -217,4 +218,41 @@ var _ = Describe("Metrics Integration", Ordered, func() {
 				))).To(Succeed())
 		})
 	})
+
+	Context("When measuring latency between integration tests and release creation", func() {
+
+		var snapshot *applicationapiv1alpha1.Snapshot
+
+		BeforeEach(func() {
+			// Mocking a snapshot with the condition for the test
+			snapshot = &applicationapiv1alpha1.Snapshot{
+				Status: applicationapiv1alpha1.SnapshotStatus{
+					Conditions: []metav1.Condition{
+						{
+							Type:               "AppStudioTestSucceeded",
+							LastTransitionTime: metav1.NewTime(time.Now().Add(-60 * time.Second)),
+						},
+					},
+				},
+			}
+		})
+
+		It("adds an observation to ReleaseLatencySeconds", func() {
+			RegisterReleaseLatency(snapshot)
+
+			// Extracting the start time from the condition
+			condition := meta.FindStatusCondition(snapshot.Status.Conditions, "AppStudioTestSucceeded")
+			startTime := condition.LastTransitionTime
+			endTime := metav1.NewTime(time.Now())
+
+			// Calculate observed latency
+			observedLatency := endTime.Sub(startTime.Time).Seconds()
+
+			// Ensure the observed latency is within a range ±5ms 60 seconds
+			Expect(observedLatency).To(BeNumerically(">=", 59.995))
+			Expect(observedLatency).To(BeNumerically("<=", 60.005))
```

Comment:
I have hard time to understand, how this part of the test is testing `RegisterReleaseLatency`, it looks like the test is testing itself only